SETUP.md

```diff
@@ -0,0 +1,41 @@
+# Setup
+
+This is a beginner friendly guide for setting up everything necessary to work on peerhaven.
+
+## Languages
+
+- [JavaScript](http://javascript.info/)
+- [Markdown](https://blog.ghost.org/markdown/)
+
+## Software
+
+We work with [elementary](https://elementary.io/).
+If parts of this guide don't work for your system, let us know by [submitting an issue](CONTRIBUTING.md#submitting-issues)!
+
+Our web UI is mainly tested with [Firefox](https://www.mozilla.org/en-US/firefox/) and [Chrome](https://www.google.com/chrome/browser/desktop/index.html).
+
+### Atom
+
+For writing, we use [Atom](https://flight-manual.atom.io/getting-started/sections/why-atom/).
+Recommended packages are:
+
+- [Atom IDE](https://ide.atom.io/)
+- [linter-js-standard](https://atom.io/packages/linter-js-standard), with _semistandard_ style selected
+- [Minimap](https://atom.io/packages/minimap) with [git-diff plugin](https://atom.io/packages/minimap-git-diff)
+- [Teletype](https://teletype.atom.io/)
+
+[Ubuntu Mono](https://fonts.google.com/specimen/Ubuntu+Mono) is a nice coding font.
```

Comment:
PROPAGANDA. *If you're looking for a nice coding font, you may want to check out Ubuntu Mono ;)